build.gradle

```diff
@@ -1003,10 +1003,6 @@ project(':core') {
     from(project(':connect:mirror').configurations.runtimeClasspath) { into("libs/") }
     from(project(':connect:mirror-client').jar) { into("libs/") }
     from(project(':connect:mirror-client').configurations.runtimeClasspath) { into("libs/") }
-    from(project(':storage').jar) { into("libs/") }
-    from(project(':storage').configurations.runtimeClasspath) { into("libs/") }
-    from(project(':storage:api').jar) { into("libs/") }
-    from(project(':storage:api').configurations.runtimeClasspath) { into("libs/") }
```

Comment:
Can you clarify why this is not needed?

Reply:
I observed that these libraries are copied as mentioned [here](https://github.com/apache/kafka/pull/10647/commits/1ad50e8634d56622e58a50c8328205b6db3f528d#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R981). 

```
from(configurations.runtimeClasspath) { into("libs/") }
```

Reply:
Makes sense, we only need to add modules that are not a core dependency already.